meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -17,4 +18,15 @@ SRC_URI:append:am62xx-evm = " \
     file://lmp.cfg \
 "
 
+PATCHTOOL = "git"
```

Comment:
Is this needed?

Reply:
it helps since we are updating file in the source tree (a dtsi file)

Reply:
Sure, but if not needed, I would prefer for it to not be added as it is yet another line we would have to maintain in the bbappend.

Reply:
by default bitbake uses the quilt for patching but it works without issues in any source tree, one diference is with quilt it don't produce commits in the source tree

Reply:
Not having git in a tree that we patch and then update with sed is confusing when working/developing on local trees. So in terms of usability I'd say we have to have this.

But now, can you explain a bit more what does the maintenance task involve? why is having this line an issue? I dont understand sorry...

Reply:
@quaresmajose forgot about that one, this will change the commit hash which is also later part of the u-boot version string, which is indeed not desirable.

Reply:
> Not having git in a tree that we patch and then update with sed is confusing when working/developing on local trees. So in terms of usability I'd say we have to have this.

For dev it makes sense, but as @quaresmajose wrote above, this will change the git version identifier as it gets applied as a commit, which will also change the hash string that is part of the u-boot version (which you can read during boot), which is not ideal (and why OE uses quilt by default, to avoid changing the underlying tree identifier).

Reply:
Another consequence of this is breaking the binary reproducibility because it will change the commit hash randomly when applying the patch. Because of this each build will have a different hash and as the hash is encoded on the u-boot version the binary is not the same.

Reply:
I don't see how this is a problem since releases are based on manifests and issues are reported based on releases and not tree commits.
But anyway, will drop.